src/Essentials/src/Xamarin.Essentials/Xamarin.Essentials.csproj

```diff
@@ -0,0 +1,125 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.0;netstandard2.0;Xamarin.iOS10;Xamarin.TVOS10;Xamarin.WatchOS10;MonoAndroid80;MonoAndroid81;MonoAndroid90;MonoAndroid10.0;tizen40;Xamarin.Mac20;</TargetFrameworks>
```

Comment:
Can you add these additional frameworks to the provisioning part of our cake script?

https://github.com/xamarin/Xamarin.Forms/blob/5.0.0/build.cake#L89

Reply:
I think it is already there: 28, 29 and 30.

Unless I am mistaken?

Reply:
Yea but we also need 26 and 27 for MonoAndroid80 and 81

When I opened the solution it told me those were missing and I needed to install them

Reply:
🤦

Reply:
https://github.com/xamarin/Xamarin.Forms/pull/13613